plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2730,6 +2730,39 @@ function wc_update_910_add_launch_your_store_tour_option() {
 	add_option( 'woocommerce_show_lys_tour', 'yes' );
 }
 
+/**
+ * Add woocommerce_hooked_blocks_version option for existing stores that are using a theme that supports the Block Hooks API
+ */
+function wc_update_920_add_wc_hooked_blocks_version_option() {
+	if ( ! wc_current_theme_is_fse_theme() && ! current_theme_supports( 'block-template-parts' ) ) {
+		return;
+	}
+
+	$option_name       = 'woocommerce_hooked_blocks_version';
+	$option_value      = get_option( $option_name );
```

Comment:
We could do but it feels redundant don't you think? Given that this is the migration we can pretty much be sure that the option doesn't exist since we only start introducing it to new stores (which wont run this migration) in this PR also?

Reply:
But then again, I do the `false` check in the conditions 🤔

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/49302/commits/00cf4adb729d118621073cfc9ec988f844d2b9db